e2e/windows/scenarios/windows/property-windows-template.json

```diff
@@ -61,7 +61,7 @@
     }
   },
   "K8sComponents": {
-    "WindowsPackageURL": "https://acs-mirror.azureedge.net/kubernetes/v1.23.12/windowszip/v1.23.12-1int.zip"
+    "WindowsPackageURL": "https://acs-mirror.azureedge.net/kubernetes/v${orchestratorVersion}/windowszip/v${orchestratorVersion}-1int.zip"
```

Comment:
can we support to test a hotfix package?
v1.25.2-hotfix.20221006/windowszip/v1.25.2-hotfix.20221006-1int.zip

Reply:
@junjiezhang1997 should we add a pipeline variable?

Reply:
Added `WINDOWS_PACKAGE_URL` (see the latest file change)